scene/gui/color_picker.cpp

```diff
@@ -174,22 +173,16 @@ void ColorPicker::_update_color() {
 	for(int i=0;i<4;i++) {
 		scroll[i]->set_max(255);
 		scroll[i]->set_step(0.01);
-		if (raw_mode_enabled && i != 3)
+		if (raw_mode_enabled) {
+			if (i == 3)
+				scroll[i]->set_max(1);
 			scroll[i]->set_val(color.components[i]);
-		else
-			scroll[i]->set_val(color.components[i]*255);
+		} else {
+			scroll[i]->set_val(color.components[i] * 255);
```

Comment:
you are doing it again, why do you multiply the compunent by 255? it's not suposed to be that way, only difference is allowing numbers avobe 1.